src/pr-info.ts

```diff
@@ -280,7 +280,7 @@ function getLastMaintainerBlessingDate(timelineItems: PR_repository_pullRequest_
 }
 
 async function getPackageInfosEtc(
-    paths: string[], headId: string, fetchFile: typeof defaultFetchFile, getDownloads: typeof getMonthlyDownloadCount
+    paths: string[], headId: string, fetchFile: typeof defaultFetchFile, getDownloads: typeof getMonthlyDownloadCount,
```

Comment:
Also not great for function arguments--?

Reply:
Reverted :heavy_check_mark: